libs/upd/components/src/lib/charts/charts.component.ts

```diff
@@ -249,21 +257,37 @@ export class ChartsComponent implements OnInit {
   }
 
   yLeftTickFormat(data: number) {
-    return (data / 1000).toLocaleString(this.lang);
+    return (data / 1000);
   }
 
   yRightTickFormat(data: number) {
-    return (data / 1000).toLocaleString(this.lang);
+    return (data / 1000);
   }
 
   xAxisTickFormat(data: number) {
-    return (data / 1000).toLocaleString(this.lang);
+    return (data / 1000);
   }
 
   xAxisPercentTickFormat(data: number) {
     return `${data * 100}%`;
   }
 
+  yLeftTickFormatFR(data: number) {
```

Comment:
Having separate methods for French formatting doesn't seem like a particularly good idea to me. You'll need to know what the current language is to know which function to call, so why not just pass it to the single formatting function?

Reply:
Don't know why I included the percentage on the axis, I just removed it